src/context/AppProvider/AppContext.tsx

```diff
@@ -155,6 +155,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
   // once portfolio is done loading, fetch fox voting power
   useEffect(() => {
+    if (!requestedAccountIds.length) return
```

Comment:
Shouldn't this be caught by `selectPortfolioLoadingStatusGranular` ?

Reply:
yeah, I just added it to the `getAllTxHistory` effect as a "doesn't hurt", and figured I would add in these two effects as well